example/demo_survey/src/survey/templates/VisitedPlacesSection.tsx

```diff
@@ -260,6 +259,7 @@ export class VisitedPlacesSection extends React.Component<any, any> {
         case 'question': component = <Question {...defaultProps} />; break;
         case 'group':    component = <Group    {...defaultProps}
           groupConfig     = {this.props.groups[widgetShortname]}
+          groupsConfig    = {this.props.groups}
```

Comment:
This is a BREAKING POINT, no? Should you mention it on the commit message?

Reply:
C'est pas exactement un breaking point, c'est juste que vu que c'était pas typé, ça ne se plaignait pas.
Mais maintenant que je suis en train de changer tout ça, il s'avère que ce groupsConfig est pas mal genre pas utilisé pentoute... Je vais le rendre optionnel sur cette PR à la place.

Reply:
J'ai parlé trop vite, pour l'instant groupsConfig est obligatoire et dans ce cas-ci, SI les visitedPlaces avaient eu des sous-groupes, ça aurait planté. Ça fixe donc un bug qui n'existe pas encore, mais pas un breaking change

Reply:
C'est bon!